atuin/src/command/client/search.rs

```diff
@@ -158,7 +158,7 @@ impl Cmd {
             settings.filter_mode = self.filter_mode.unwrap();
         }
         if self.inline_height.is_some() {
-            settings.inline_height = self.inline_height.unwrap();
+            settings.display.inline_height = self.inline_height.unwrap();
```

Comment:
Any display settings are now a level deeper because of the new settings type, even though it's not visible externally in config.